src/components/AttestedClaimVerificationView/AttestedClaimVerificationView.tsx

```diff
@@ -88,7 +87,9 @@ class AttestedClaimVerificationView extends React.Component<Props, State> {
     return (
       <div className="attributes">
         {propertyNames.map((propertyName: string) => {
-          const propertyTitle = this.getCtypePropertyTitle(propertyName)
+          const propertyTitle = this.props.cType
+            ? getCtypePropertyTitle(propertyName, this.props.cType)
+            : propertyName
```

Comment:
Moved existence checks to the calling function, getCtypePropertyTitle now always has to take a ctype.